frameworks/PHP/ubiquity/benchmark_config.json

```diff
@@ -26,26 +26,28 @@
         "versus": "php"
       },
       "raw": {
-            "db_url": "/Raw",
-            "query_url": "/Raw/query/",
-            "fortune_url": "/RawFortunes",
-            "update_url": "/Raw/update/",
-            "port": 8080,
-            "approach": "Realistic",
-            "classification": "Fullstack",
-            "database": "MySQL",
-            "framework": "Ubiquity",
-            "language": "PHP",
-            "flavor": "PHP7",
-            "orm": "Raw",
-            "platform": "php-fpm",
-            "webserver": "nginx",
-            "os": "Linux",
-            "database_os": "Linux",
-            "display_name": "ubiquity-raw",
-            "notes": "",
-            "versus": "php"
-          },
+        "json_url": "/Json",
+        "plaintext_url": "/Plaintext",
```

Comment:
Please remove the json and plaintext.
Are the same that with ORM.

Reply:
The framework configuration is slightly different, including for `Plaintext` and `Json`:
The configuration does not need to start the ORM cache in this raw bench.

Reply:
>For me that is an arquitectural problem.

I'd rather say it's code factorization.

In all tests except `Raw`, the ORM cache is started in the file `services.php`, used in all requests.

If I start the ORM cache only in the Db and Fortunes controllers (it is possible), I am forced to duplicate all the code for Swoole, for which ORM starts only once, and not at each request...

By adding the `Plaintext` and `Json` URLs to the `Raw` bench,
I can now evaluate these 2 tests without unnecessary cache, and without the duplication due to Swoole :-)

Reply:
so do I leave it unchanged or do I correct, by duplicating the `Db` and `Fortunes` controllers for `Swoole`?

Reply:
Why do you create new controllers for the raw version and not for swoole ?

I will show you some disadvantages:

1. Every run in the continuous bench, need to run 2 extra tests . And add extra noise to the visualize.
2. Almost nobody will understand the json and plaintext with and without ORM.
3. You are showing a bad example of how to use your framework.

Reply:
It's okay, @joanhey .
I close this PR and will open another one to correct.